dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'zendesk'
-version: '0.6.0'
+version: '0.6.1'
```

Comment:
Since we are going to be making a lot of changes (one of them being a breaking change) I think it would probably be best to include all of these updates into a new `v0.7.0` branch. Would you be able to update this and rebase to a release branch for `v0.7.latest`?

Sorry I should have specified the breaking change before 🤦